language/llama2-70b/evaluate-accuracy.py

```diff
@@ -5,6 +5,7 @@
 import numpy as np
 import json
 
+ACC_TARGET = {"rouge1": 43.88, "rouge2": 21.7108, "rougeL": 28.2502}
```

Comment:
I am a bit hesitant to put the ACC_TARGET here instead of importing from the submission_checker.py.